docs/experimental/ticket-auction.md

```diff
@@ -0,0 +1,258 @@
+         --burn-cap 1
+
+New contract KT1QQukCBULzFu6samB5FpbLNBXmNzArSpTs originated.
+
+```
+## NFT Auction Contract
+
+A separate NFT auction contract can be used to auction off ones NFTs to prospective buyers. We use a Dutch, descending price auction, in which the asking price of the NFT is decreased until a buyer is found. The NFT auction contract has an entrypoint to `configure` the auction by customizing
+- the `opening_price`
+- the `reserve_price`-- the minimum price the NFT can be reduced to
+- the minimum `start_time` of the auction
+- the `round_time`-- the minimum time the item stays at the same price before its asking price can be decreased.
+- the `ticket`-- the actual NFT that will be auctioned off  
+
+The auction also has entrypoints to `start` the auction, `drop_price` of the NFT after a round has passed, `buy` the NFT by sending the asking price, and an entrypoint for the admin to `cancel` the auction and send the NFT back to their wallet.  
+
+### Auction
+
+Alice originates nft-auction contract
+
+```sh
+$ tezos-client originate contract nft-auction transferring 0 from alice \
+        running "$(pwd)/michelson/nft_auction.tz" \
+         --dry-run \
+         --init "Pair (Pair \"tz1LrQB9HrJcUaD9NKEvV65tnaiU8trPXwmt\" (Pair 0 (Pair 0 (Pair False ( Pair 0 0))))) {}" \
+         --burn-cap 1
+
+         { parameter
+                     (or (or (or (contract %buy (ticket nat)) (contract %cancel (ticket nat)))
+                             (or (pair %configure
+                                    (nat %opening_price)
+                                    (pair (nat %set_reserve_price)
+                                          (pair (timestamp %set_start_time) (pair (int %set_round_time) (ticket %ticket nat)))))
+                                 (nat %drop_price)))
+                         (unit %start)) ;
+                   storage
+                     (pair (pair %data
+                              (address %admin)
+                              (pair (nat %current_price)
+                                    (pair (nat %reserve_price)
+                                          (pair (bool %in_progress) (pair (timestamp %start_time) (int %round_time))))))
+                           (big_map %tickets nat (ticket nat))) ;
+                   code {...}
+
+New contract KT1HWaMyNmjVGMBPUSm3QxJnFRLi4LQJi1tG originated.
+```
+## Demo
+Now that the wallet and auction contracts are originated Alice can create an NFT and auction it off. In the auction, no one will purchase the item at Alice's `opening_price` of 100tz so she drops the price to 90tz and Bob purchases the item.
+
+### Mint NFT
+
+Alice mints herself a ticket based nft with metadata
+
+```sh
+$ tezos-client transfer 0 from alice to nft-wallet
+          --entrypoint "mint" \
+          --arg "{Elt \"\" 0x68747470733a2f2f6769746875622e636f6d2f747174657a6f732f7469636b65742d7475746f7269616c732f747265652f6d61696e2f7475746f7269616c732f61756374696f6e2f6c69676f}" \
```

Comment:
I don't see any explanation of what this is. Again, following FA1.2 Lorentz style here would be helpful - use env vars (for shorter commands and additional context provided by variable name) and detailed explanation of what is what here